Rendera/include/scene.hpp

```diff
@@ -54,21 +55,25 @@ color lighting(Material mat, ray cam_ray, ray normal, vec3 point, vec3 light_src
 
 class Scene{
     public:
-        int n_sphere;
+        int n_sphere = 1;
```

Comment:
These two variables (`n_sphere` and `n_cube`) shouldn't be hard-coded in the `Scene` class.